dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/appmanager/DefaultAppManager.java

```diff
@@ -328,14 +322,12 @@ public void reloadApps() {
         .filter(app -> !exists(app.getKey()))
         .forEach(this::installApp);
 
-    jCloudsAppStorageService.discoverInstalledApps().values().stream()
-        .filter(app -> !exists(app.getKey()))
-        .forEach(this::installApp);
+    jCloudsAppStorageService.discoverInstalledApps().values().forEach(this::installApp);
```

Comment:
FYI: I effectively removed the filtering to only install what isn't already installed so that jclouds would win over local store.